src/System.Net.WebSockets.Client/tests/SendReceiveTest.cs

```diff
@@ -16,11 +16,7 @@ namespace System.Net.WebSockets.Client.Tests
 {
     public class SendReceiveTest : ClientWebSocketTestBase
     {
-        // Windows 10 Insider Preview Build 16215 introduced the necessary APIs for the UAP version of
-        // ClientWebSocket.ReceiveAsync to consume partial message data as it arrives, without having to wait
-        // for "end of message" to be signaled.
-        public static bool PartialMessagesSupported =>
-            !PlatformDetection.IsUap || PlatformDetection.IsWindows10InsiderPreviewBuild16215OrGreater;
+        public static bool PartialMessagesSupported { get; } = PlatformDetection.ClientWebSocketPartialMessagesSupported;
```

Comment:
> We don't need to allocate memory for intermediate _somePropertyBackingField, and directly refer to the property itself. So I think it's better.

I've not looked at what the property it's calling to does.  If that property is already caching the result, then I'd agree.  If it's not, then I wouldn't.